tests/unit/modules/test_ansiblegate.py

```diff
@@ -137,3 +139,42 @@ def test_virtual_function(self):
         '''
         with patch('salt.modules.ansiblegate.ansible', None):
             assert ansible.__virtual__() == 'ansible'
+
+    def test_ansible_module_call(self):
```

Comment:
This is failing on Ubuntu 16.04 py3 right now (totally unsure if its actually failing or if the Jenkins build itself has issues). I'll take a look at this later today, but otherwise everything else looks very good! Thanks for providing tests.

`08:51:43         --------  Failed Tests  -------------------------------------------------------
08:51:43           -> unit.modules.test_ansiblegate.AnsiblegateTestCase.test_ansible_module_call  
08:51:43        Traceback (most recent call last):
08:51:43          File "/tmp/kitchen/testing/tests/unit/modules/test_ansiblegate.py", line 175, in test_ansible_module_call
08:51:43            proc.assert_any_call(['echo', '{"ANSIBLE_MODULE_ARGS": {"kwarg1": "foobar", "_raw_params": "arg_1"}}'], stdout=-1, timeout=1200)
08:51:43          File "/usr/local/lib/python3.5/dist-packages/mock/mock.py", line 1010, in assert_any_call
08:51:43            ), cause)
08:51:43          File "<string>", line 3, in raise_from
08:51:43        AssertionError: mock(['echo', '{"ANSIBLE_MODULE_ARGS": {"kwarg1": "foobar", "_raw_params": "arg_1"}}'], stdout=-1, timeout=1200) call not found`

Reply:
@xeacott This is now fixed. Hopefully no more failures on Ubuntu :smile: